### PR TITLE
Improve docs

### PR DESCRIPTION
**Pull request by pombredanne**
This PR refines the README documentation.

### CODE REVIEW

**Review by sbs2001 — CHANGES_REQUESTED** [review]
Thanks ! This is better, please  correct the typos I've highlighted, and we're good to merge this.

**Comment by pombredanne** [comment]
> Thanks ! This is better, please correct the typos I've highlighted, and we're good to merge this.

I could not see comments about typos, may be you did not submit your review yet and the comments are still pending the full review submissian?

File: README.rst [hunk 1]
```diff
@@ -1,95 +1,115 @@
...
+and package repositories, FOSS projects, GitHub and more. Thanks to this
+approach, the data is focused on specific ecosystems yet aggregated in a single
+database that enables querying a richer graph of relations between multiple
+incarnations of a package. Being specific increases the accuracy and validity
+of the data as the same version of an upstream package across different
+ecosystems may or may not be vulnerable to the same vulnerability.
+
+The packages are identified using Package URL `PURL 
+<https://github.com/package-url/purl-spec>`__ as primary identifiers rather than
+CPEs. This makes packeg queries more dpscific and more accurate to answer
```
Comment (sbs2001) [inline README.rst:62]: This line contains 2 typos

**Review by sbs2001 — CHANGES_REQUESTED** [review]

**Comment by sbs2001** [comment]
@pombredanne  I don't know what happened, anyways I've commented the line with typos

**Comment by sbs2001** [comment]
I commited typo fixes and merging this.

**Pull request merged**